src/views/Editor.vue

```diff
@@ -290,17 +296,49 @@ export default defineComponent({
         class="px-3 py-2 border rounded-lg mb-5 min-h-[200px]"
         :body="proposal.body"
       />
-
-      <div v-else class="s-base mb-3">
-        <div class="s-label" v-text="'Description'" />
-        <textarea v-model="proposal.body" maxlength="9600" class="s-input mb-3 h-[200px]" />
-        <SIString
-          :key="proposalKey || ''"
-          v-model="proposal.discussion"
-          :definition="DISCUSSION_DEFINITION"
-          :error="formErrors.discussion"
-        />
-        <Preview :key="proposalKey || ''" :url="proposal.discussion" />
+      <div v-else>
+        <div class="flex justify-end gap-1 py-1 px-2 border rounded-t-lg">
```

Comment:
We can give a bit more space here:
```suggestion
        <div class="flex justify-end gap-1 py-2 px-3 border rounded-t-lg">
```